code/modules/clothing/head/helmet.dm

```diff
@@ -167,3 +167,77 @@
 		set_light(2, 2, COLOR_LIGHTING_ORANGE_MACHINERY)
 	update_wear_icon()
 	..()
+
+// S E R B I A //
+
+/obj/item/clothing/head/helmet/steelpot
+	name = "steelpot helmet"
+	desc = " Titanium helmet of serbian origin. Still widely used despite of its discontinued production."
+	icon_state = "steelpot"
+	armor = list(melee = 25, bullet = 30, energy = 0, bomb = 25, bio = 0, rad = 0) // same as IHS helmet
```

Comment:
this is worse than the handmade helmet which is 35 melee 30 bullet 30 energy. You might want to swap the armor on both of these in this PR or make it the same as the IH helmet.

Also, the comment is wrong. IH helmet has 35 melee, 30 bullet/energy proct.

Reply:
requested changes were made